src/tools/pontoon/adding_new_locale.md

```diff
@@ -105,4 +105,14 @@ Population: 1863468
 
 Find the locale code in [CLDR territoryInfo.json](https://github.com/unicode-cldr/cldr-core/blob/master/supplemental/territoryInfo.json) and multiply its `_populationPercent` with the territory `_population`. Repeat if multiple occurrences of locale code exist and sum products.
 
-At this point click **SAVE** in the bottom right corner to save and create the new locale. The "Terminology" project should appear automatically under the new team's page — in this case at https://pontoon.mozilla.org/am/
+At this point click **SAVE** in the bottom right corner to save and create the new locale. The "Terminology" project should appear automatically under the new team’s page — in this case at https://pontoon.mozilla.org/am/
+
+### Access key localization
```

Comment:
@mathjazz 
The checkbox and description currently disagree on how to spell access key. I personally suggest to use the one with a space.

Reply:
Opened a PR to fix the auto-generated spelling:
https://github.com/mozilla/pontoon/pull/2857